lib/Runtime/Library/RuntimeLibraryPch.h

```diff
@@ -99,4 +99,9 @@
 #define JS_DIAG_INLINE inline
 #else
 #define JS_DIAG_INLINE
-#endif
\ No newline at end of file
+#endif
+
+// Include this last. GNU redefines __try as `try`
```

Comment:
One more reason not to mess with `__try`. I think the previous `#if ... #else ...` pattern is clear enough. Or we could `#ifdef` out the `__try/__except` part individually. I'd support leaving it inconvenient to discourage more SEH use.